concord-ethrpc image does not build

The `concord-ethrpc` will not build on my OS X machine. It appears that a Checkstyle rule is failing.

A full stack trace is below

```
Step 10/17 : RUN mvn -f ethrpc/pom.xml package
 ---> Running in 199a68a895c7
[INFO] Scanning for projects...
[INFO] 
[INFO] -----------------< com.vmware.concord:concord-ethrpc >------------------
[INFO] Building concord-ethrpc 1.0-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- maven-checkstyle-plugin:3.0.0:checkstyle (validate) @ concord-ethrpc ---
/ethrpc/src/main/java/com/vmware/concord/ethrpc/Constants.java:91:40: expecting IDENT, found '+'
/ethrpc/src/main/java/com/vmware/concord/ethrpc/Constants.java:95:25: unexpected token: String
/ethrpc/src/main/java/com/vmware/concord/ethrpc/Constants.java:96:25: unexpected token: String
/ethrpc/src/main/java/com/vmware/concord/ethrpc/Constants.java:97:25: unexpected token: String
/ethrpc/src/main/java/com/vmware/concord/ethrpc/Constants.java:98:25: unexpected token: String
/ethrpc/src/main/java/com/vmware/concord/ethrpc/Constants.java:99:25: unexpected token: String
/ethrpc/src/main/java/com/vmware/concord/ethrpc/Constants.java:100:25: unexpected token: String
/ethrpc/src/main/java/com/vmware/concord/ethrpc/Constants.java:101:25: unexpected token: String
/ethrpc/src/main/java/com/vmware/concord/ethrpc/Constants.java:102:25: unexpected token: String
/ethrpc/src/main/java/com/vmware/concord/ethrpc/Constants.java:103:25: unexpected token: String
/ethrpc/src/main/java/com/vmware/concord/ethrpc/Constants.java:104:25: unexpected token: String
/ethrpc/src/main/java/com/vmware/concord/ethrpc/Constants.java:105:25: unexpected token: String
/ethrpc/src/main/java/com/vmware/concord/ethrpc/Constants.java:106:25: unexpected token: String
/ethrpc/src/main/java/com/vmware/concord/ethrpc/Constants.java:107:25: unexpected token: String
/ethrpc/src/main/java/com/vmware/concord/ethrpc/Constants.java:108:25: unexpected token: String
/ethrpc/src/main/java/com/vmware/concord/ethrpc/Constants.java:109:25: unexpected token: String
/ethrpc/src/main/java/com/vmware/concord/ethrpc/Constants.java:110:25: unexpected token: String
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  3.503 s
[INFO] Finished at: 2020-04-26T21:44:03Z
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-checkstyle-plugin:3.0.0:checkstyle (validate) on project concord-ethrpc: An error has occurred in Checkstyle report generation.: Failed during checkstyle configuration: Exception was thrown while processing /ethrpc/src/main/java/com/vmware/concord/ethrpc/Constants.java: MismatchedTokenException occurred during the analysis of file /ethrpc/src/main/java/com/vmware/concord/ethrpc/Constants.java. expecting EOF, found '}' -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException

```

## Steps to reproduce

`./docker/build_images.sh`

## Context

 - Used versions (Concord version): latest

## Deliverables

- It should create the concord-ethrpc image

--- a/tools/ethrpc/src/main/java/com/vmware/concord/ethrpc/Constants.java
+++ b/tools/ethrpc/src/main/java/com/vmware/concord/ethrpc/Constants.java
@@ -87,7 +87,7 @@ public class Constants {
                                        + "0x4AffeC47CB121aF0e9fd5915895996e3f8Ad3A4A,"
                                        + "0x0eDDA8153B03c40fA3A40cC07e28089DD0435464,"
                                        + "0x06e20BB182FC80AA16903a565CC4f93e2b47D423,"
-                                       + "0xfD444ECB2c868C3A851cE6389c24dbCB4Eb665fd,",
+                                       + "0xfD444ECB2c868C3A851cE6389c24dbCB4Eb665fd,"
                                        + "0xafCE1Bf29cE631B0AeE33308Ff6A8b37586b3034,"
                                        + "0x4461d0d1e80073b3490398af1252F0B925e42A8F,"
                                        + "0xe1c7a0aDE49A2B7D8788bfFb1438F722A0bE925F,"

